Formula/ansible-lint.rb

```diff
@@ -124,6 +86,10 @@ class AnsibleLint < Formula
 
   def install
     virtualenv_install_with_resources
+    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    site_packages = "libexec/lib/python#{xy}/site-packages"
+    ansible = Formula["ansible"].opt_prefix
```

Comment:
```suggestion
    site_packages = "lib/python#{xy}/site-packages"
    ansible = Formula["ansible"].opt_libexec
```

Reply:
I will use `Formula["ansible"].opt_libexec` and `libexec` instead of `prefix` for `ansible-lint`'s path

Reply:
Whoops, nope, this doesn't work.

Reply:
Worked!